1st homework/exer8.py

```diff
@@ -0,0 +1,7 @@
+n = int(input("Введите число n: "))
+m = int(input("Введите число m: "))
+k = int(input("Введите число k: "))
+if k%n == 0 or k%m == 0:
```

Comment:
```suggestion
if k % n == 0 or k % m == 0:
```
Плюс у вас не хватает одного обязательного условия. Подумайте какого?

Решение рассмотрим на ближайшем семинаре.